src/journal/journalctl.c

```diff
@@ -89,6 +89,7 @@ static bool arg_all = false;
 static PagerFlags arg_pager_flags = 0;
 static int arg_lines = ARG_LINES_DEFAULT;
 static bool arg_no_tail = false;
+static bool arg_truncate = false;
```

Comment:
rename to arg_truncate_newline, i.e. please stick to the same name all throught the codebase